lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -248,6 +248,59 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		await driverHelper.clickWhenClickable( this.driver, By.css( '.dashicons-no-alt' ) );
 	}
 
+	async insertPaymentButtonDetails( {
```

Comment:
Should this function be in its own block component - this file is getting pretty big - there's probably some other functions that should be split out also

Reply:
I made a separate component for payment block - `payment-block-component.js` and added requested changes there.